ui/page/tickets/overview.go

```diff
@@ -132,20 +140,28 @@ func (pg *Page) loadPageData() {
 }
 
 func (pg *Page) Layout(gtx layout.Context) layout.Dimensions {
-	return components.UniformPadding(gtx, func(gtx layout.Context) layout.Dimensions {
-		sections := []func(gtx C) D{
-			func(ctx layout.Context) layout.Dimensions {
+
+	widgets := []layout.Widget{
+		func(ctx layout.Context) layout.Dimensions {
+			return components.UniformHorizontalPadding(gtx, func(gtx layout.Context) layout.Dimensions {
 				return pg.ticketPriceSection(gtx)
-			},
-			func(ctx layout.Context) layout.Dimensions {
+			})
+		},
+		func(ctx layout.Context) layout.Dimensions {
+			return components.UniformHorizontalPadding(gtx, func(gtx layout.Context) layout.Dimensions {
 				return pg.ticketsLiveSection(gtx)
-			},
-			func(ctx layout.Context) layout.Dimensions {
+			})
+		},
+		func(ctx layout.Context) layout.Dimensions {
+			return components.UniformHorizontalPadding(gtx, func(gtx layout.Context) layout.Dimensions {
 				return pg.stakingRecordSection(gtx)
-			},
-		}
-		return pg.ticketPageContainer.Layout(gtx, len(sections), func(gtx C, i int) D {
-			return sections[i](gtx)
+			})
+		},
+	}
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.Inset{Top: values.MarginPadding24}.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+		return material.List(pg.Theme.Base, pg.list).Layout(gtx, len(widgets), func(gtx C, i int) D {
```

Comment:
Why use gio material directly? Kindly add this as a file on the decredmaterial package. list.go.
See here for examples.
https://github.com/planetdecred/godcr/blob/master/ui/decredmaterial/checkbox.go.
https://github.com/planetdecred/godcr/blob/master/ui/decredmaterial/label.go.